platform-emf/src/main/resources/com/softicar/platform/emf/emf-form-style.css

```diff
@@ -14,16 +14,33 @@
 	border-radius: var(--BORDER_RADIUS);
 	box-shadow: var(--BOX_SHADOW_OVERLAY);
 	display: inline-grid;
+	grid-auto-columns: max-content;
+	align-self: start;
 	outline-style: none;
+	gap: 0px;
```

Comment:
Why is this necessary, all of a sudden, and why won't things look gapless now?

Reply:
This is necessary to make the EmfFormFrame have the same dimensions and gaps as an EmfFormPopup.
EmfFormFrame has 2 Components, to the 5px gap would make a frame seem bigger. See screenshot
![image](https://user-images.githubusercontent.com/90852097/187672982-093f74b4-cb59-421d-ae7d-128de8e0112a.png)